include/lbann/utils/cudnn.hpp

```diff
@@ -84,6 +84,23 @@ class Layer;
 
 namespace cudnn {
 
+template <typename T>
+struct ScalingParameterT
+{
+  using type = T;
+};
+
+template <typename T>
+using ParamType = typename ScalingParameterT<T>::type;
```

Comment:
We should document and link to the [cuDNN docs](https://docs.nvidia.com/deeplearning/sdk/cudnn-archived/cudnn_701/cudnn-user-guide/index.html#scaling-parameters).